syftlib/src/main/java/org/openmined/syft/interfaces/ProcessSchedulers.kt

```diff
@@ -0,0 +1,18 @@
+package org.openmined.syft.interfaces
```

Comment:
Just a small thing to finish:
Could you put this in a separate package? Something like `syft.process`?
Also, `SignallingInterface` should go with `SignalligClient`. Maybe a package `syft.signalingclient`?

Reply:
I think we should decide the overall folder structure